src/app/tests/TestStatusIB.cpp

```diff
@@ -163,19 +163,19 @@ TEST_F(TestStatusIB, ConversionsFromClusterStatusCodeWork)
     StatusIB successWithCode{ ClusterStatusCode::ClusterSpecificSuccess(123u) };
     EXPECT_EQ(successWithCode.mStatus, Status::Success);
     EXPECT_TRUE(successWithCode.IsSuccess());
-    ASSERT_TRUE(successWithCode.mClusterStatus.HasValue());
-    EXPECT_EQ(successWithCode.mClusterStatus.Value(), 123u);
+    ASSERT_TRUE(successWithCode.mClusterStatus.has_value());
+    EXPECT_EQ(successWithCode.mClusterStatus.value(), 123u); // NOLINT(bugprone-unchecked-optional-access)
```

Comment:
nit: maybe make it consistent between using `*` vs `.value()` ?

```suggestion
    EXPECT_EQ(*successWithCode.mClusterStatus, 123u); // NOLINT(bugprone-unchecked-optional-access)
```